### ISSUE TITLE
Pour le mode prod avec diffent groupe scout: Même code sur different sous domaine.

### ISSUE THREAD

**Issue by nuffer** [sysmoh/intranetBS#58]
http://stackoverflow.com/questions/12667282/symfony-2-multiple-apps

**Comment by ozwin** [comment]
Oui... et non. Je me demande si c'est pas mieux d'avoir une instance par sous-domaine. Avec un bon process d'intgération continue c'est très facile à maintenir, et au moins on migre (et résout les problèmes) que quand c'est nécessaire. S'il faut faire du dirty tuning, ça reste possible sur une instance.

**Comment by nuffer** [comment]
C'est toi le bosse pour cette question. il faut choisir la solution la plus facile à maintenir!!!!
